packages/truffle-box/lib/config.js

```diff
@@ -1,9 +1,7 @@
-var fs = require("fs-extra");
+const fs = require("fs-extra");
```

Comment:
How do you feel about making all instances of the fs-extra library set to `fse` (the docs recommend this as an alternative if you don't want to use the variable fs)? I like making it clear that it is something different from the classic `fs` library but I have no problem with just leaving it as `fs` as well.

Reply:
Looks like we use both (`fs & `fs-extra`) throughout the codebase, so makes sense to differentiate.

Reply:
Yeah, I mean the docs for `fs-extra` say that it is meant as a "drop in" replacement for `fs`. I'm always into the idea of making things extra clear.